lib/EntryModuleNotFoundError.js

```diff
@@ -2,15 +2,20 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-function EntryModuleNotFoundError(err) {
-	Error.call(this);
-	this.name = "EntryModuleNotFoundError";
-	this.message = "Entry module not found: " + err;
-	this.details = err.details;
-	this.error = err;
-	Error.captureStackTrace(this, EntryModuleNotFoundError);
+"use strict";
+
+class EntryModuleNotFoundError extends Error {
+	constructor(err) {
+		super();
+
+		if(Error.hasOwnProperty("captureStackTrace")) {
```

Comment:
captureStackTrace should be at the end of the constructor. (It must be after settings `message` to grab it correctly)